tests/fixtures.py

```diff
@@ -50,10 +50,18 @@ def generate_board_and_error_queues(num_boards: int = 4, queue_type=MPQueue):
     return board_queues, error_queue
 
 
-def get_generic_base64_args(recording_directory: Optional[str] = None) -> str:
+def get_generic_base64_args(
+    recording_directory: Optional[str] = None, analysis_output_dir: Optional[str] = None
+) -> str:
     if not recording_directory:
         recording_directory = get_current_file_abs_directory()
-    test_dict = {"log_file_id": "any", "recording_directory": recording_directory}
+    if not analysis_output_dir:
+        analysis_output_dir = get_current_file_abs_directory()
```

Comment:
We probably want to make these file dirs different just in case a test tries to use both of them. Could just use
```
recording_directory = os.path.join(get_current_file_abs_directory(), "recordings")
analysis_output_dir = os.path.join(get_current_file_abs_directory(), "time_force_data")
```